src/core/catalog.js

```diff
@@ -638,10 +639,39 @@ class Catalog {
       return { name: stringToPDFString(nestedName), order: nestedOrder };
     }
 
+    function parseRBGroups(rbgrps) {
+      if (!Array.isArray(rbgrps)) {
+        return null;
+      }
+
+      // iterate over RB groups (literal arrays or refs thereof)
+      for (const value of rbgrps) {
+        const rbGroup = value instanceof Ref ? xref.fetchIfRef(value) : value;
+        // ignore wrong (non-array) refs and empty arrays
```

Comment:
This comment seems a bit redundant here, since it's quite obvious what the next lines does.

Reply:
Ok, will delete this comment.